packages/client/components/DashNavList/DashNavList.tsx

```diff
@@ -63,7 +64,9 @@ const DashNavList = (props: Props) => {
             </Tooltip>
           </div>
           <DashNavListTeams onClick={onClick} organizationRef={org} />
-          {org.viewerTeams.length === 0 && <EmptyTeams organizationRef={org} />}
+          {org.teams.filter((team) => team.isViewerOnTeam).length === 0 && (
```

Comment:
+1 this should be a `!org.teams.some((team) => team.isViewerOnTeam)` to have an early return.